frontend/src/pages/NewMetadataType/NewMetadataTypeBody.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Multiselect from 'react-widgets/lib/Multiselect';
+import 'react-widgets/dist/css/react-widgets.css';
+import { Redirect } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+import { alertActions } from '../../state/actions/alert';
+import { useGetTags } from '../../sharedComponents/hooks/GetTags';
+
+const Wrapper = styled.form`
+  flex: 1;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-content: space-around;
+`;
+
+const Input = styled.input`
+  flex: 0 0 2em;
+  margin: 0.5em 1em;
+`;
+
+const TextArea = styled.textarea`
+  flex: 0 0 10em;
+  margin: 0.5em 1em;
+`;
+
+const Select = styled(Multiselect)`
+  & > .rw-widget-picker > div {
+    display: flex;
+    align-items: center;
+    & > ul > li {
+      margin-top 0px;
+    }
+  }
+`;
+
+export const NewMetadataTypeBody = () => {
+  const allTags = useGetTags();
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [tags, setTags] = useState([]);
+  const [submissionStatus, setSubmissionStatus] = useState('');
+
+  const dispatch = useDispatch();
+
+  const submit = async (e) => {
+    e.preventDefault();
+    try {
+      const res = await fetch('/api/MetadataType', {
+        method: 'PUT',
+        body: JSON.stringify({
+          name,
+          description,
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+      const { ok, status } = res;
+      if (!ok) {
+        const err = new Error();
+        err.status = status;
+        err.request = 'initial';
+        throw err;
+      }
+      const tagReses = await Promise.all(tags.map((tag) => (
+        fetch(`/api/MetadataType/${name}/tag`, {
+          method: 'PUT',
+          body: JSON.stringify({
+            name: tag,
+          }),
+          headers: {
+            'Content-Type': 'application/json',
```

Comment:
Might as well just add the Authorization header in this branch? as it is now it doesn't work as the backend requires authentication.